src/content/docs/agents/net-agent/getting-started/introduction-new-relic-net.mdx

```diff
@@ -39,6 +39,32 @@ With New Relic's support for .NET, you can monitor your apps in dynamic or distr
 * Self-hosted Windows and Linux systems
 * Amazon AWS EC2 VMs
 
+## Install the agent on Windows with our guided install [#guided-install]
+
+Our guided install creates a customized CLI command for your environment that downloads and installs the .NET agent for Windows IIS applications. 
+
+Ready to get started? Click one of these buttons to try it out.
+
```

Comment:
Ditto for the "Click one" comment I had for Java, I wonder if there's clearer language we can use about which button.

Reply:
I'll think on it.